lib/Messenger/Handler/AssetUpdateTasksHandler.php

```
@@ -33,6 +33,9 @@ public function __construct(protected LoggerInterface $logger)
     public function __invoke(AssetUpdateTasksMessage $message)
     {
         $asset = Asset::getById($message->getId());
+        if (!$asset) {
+            return;
```

Should we also log some message here? as it will be silently ignored in the handler.

---

I add a debug log.